app/classifier/tasks/survey/utility.js

```diff
@@ -0,0 +1,12 @@
+function getQuestionIDs(task, choiceID) {
+  if (task.choices[choiceID].noQuestions) {
+    return [];
+  }
+  if (!(task.questionsMap && Object.keys(task.questionsMap).indexOf(choiceID) >= 0)) {
+    return task.questionsOrder;
+  }
+  return task.questionsMap[choiceID];
+}
+
+export default getQuestionIDs;
+export { getQuestionIDs };
```

Comment:
You don't need to export getQuestionIds twice here. I think `export default { getQuestionIDs }` should work.